test/ex_doc/autolink_test.exs

```diff
@@ -449,6 +449,20 @@ defmodule ExDoc.AutolinkTest do
     assert_unchanged(~m"`Unknown`")
 
     assert_unchanged(~m"[Blank](about:blank)")
+
+    # warnings with functions starting with an underscore
+    # TODO in a following commit: Split this test into tests inside a describe block
```

Comment:
Other than that, everything looks good to me ^^

Reply:
That is intentional. I did' t want the reviewing processes be swamped with unrelated changes. I would leave it in this PR, and in a next commit split it into tests.

Reply:
Alright, sounds good!